docs/SETUP.md

```diff
@@ -458,3 +458,98 @@ You can check the logs with this command
 ```
 journalctl -u ilp-kit -f -n 500
 ```
+## Run with Forever
+This is alternative way of systemd setup. 
+
+Install "forever" with the command, if you don't have "forever" package yet. 
+``` 
+sudo npm install forever -g
+```
+
+Create a site.sh file 
+```
+nano site.sh
+```
+
+with the contents
+```
+#!/bin/bash
+
+start() {
+    echo -n "Starting forever: "
+	forever start  --workingDir=/var/ilpkit  -c "npm start" -a --uid ledger /var/ilpkit
+    return 0
+}
+
+
+stop() {
+   forever stop 0
+    killall -9 node
+    return 0
+}
+
+log() {
+forever logs  0 -f
+return 0
+}
+
+update(){
+ git add *
+ git commit -a -m "local file server commit message"
+ git fetch --all
+ git fetch origin master
+ git merge -s recursive -X theirs origin/master
+}
+
+case "$1" in
+  start)
+    stop
+    start
+    ;;
+  restart)
+    stop
+    start
+    ;;
+  update)
+     stop
+     update
+     start
+     ;;
+    log)
+      log
+	;;
+  stop)
+    stop
+    ;;
+   *)
+    echo $"Usage: $0 {update|start|stop|restart|log}"
+    exit 2
+
+esac
+```
+change where `/var/ilpkit` is the path to your real `ilp-kit` folder.
```

Comment:
"Change `/var/ilpkit` to your ILP kit's working directory, (e.g. `/home/$USER/ilp-kit`)". The default should probably be `/home/$USER/ilp-kit`, because that's what's used in the instructions.